src/contexts/theme.context.js

```diff
@@ -0,0 +1,24 @@
+const { createContext, useState, useContext } = require("react");
+
+const ThemeContext = createContext()
+
+export const ThemeProvider = ({ children }) => {
+    const [theme, setTheme] = useState('light')
+
+    function toggleTheme() {
+        theme === 'light' ? setTheme('dark') : setTheme('light')
+    }
```

Comment:
appreciation - it is good that you have added dark theme to your app.

suggestion - you can add lighter shade of the colors for dark theme, it will look more better.

Reply:
yes, agreed. I'll work on it.